src/test/java/com/google/sps/servlets/UserDataServletTest.java

```diff
@@ -102,7 +120,7 @@ public void testGetMethodWithInvalidId() throws Exception {
     when(mockResponse.getWriter()).thenReturn(writer);
 
     // Test the doGet method by calling the wrapper method
```

Comment:
nit: update comment along with all other instances in this file (should no longer refer to "wrapper")